src/localisation/string_ids.h

```diff
@@ -2441,6 +2441,8 @@ enum {
 	STR_CHEAT_DISABLE_LITTERING_TIP = 5802,
 
 	STR_SHORTCUT_MUTE_SOUND = 5804,
+	
+	STR_SERVER_TIP_ADVERTISE = 5805,
```

Comment:
Done, but also.. ooh..
Look at Lines: 2062, 2074, 2428->2438
They don't have tip in the end, should that be changed?